### PR TITLE
feat: indicate in metadata that the quickstart option was used in project creation

### PR DESCRIPTION
**Pull request by svirs**
### Description

Project metadata.template should reflect if the --quickstart flag was used

### Testing

Project metadata should show `cli-quickstart` rather than `cli-clean` if the --quickstart flag was used for pulling down a project and its schema

### CODE REVIEW

File: packages/@sanity/cli/src/actions/init-project/initProject.ts [hunk 1]
```diff
@@ -963,10 +963,9 @@ export default async function initSanity(
   }
 
   function selectProjectTemplate() {
-    // Make sure the --quickstart and --template are not used together
-    // Force template to clean if --quickstart is used
+    // --quickstart should override template selection
     if (flags.quickstart) {
-      return 'clean'
+      return 'quickstart'
```
Comment (RostiMelk) [inline packages/@sanity/cli/src/actions/init-project/initProject.ts:968]: This is going to break as there is no template called "quickstart". 
The `--quickstart `flag relies on the "clean" template, and modifies the template with its remote schema.

**Review by RostiMelk — CHANGES_REQUESTED** [review]
I suggest looking into how we import/get a template here:
https://github.com/sanity-io/sanity/blob/174a61641312734ca29d57803119ca3dfad0b78e/packages/@sanity/cli/src/actions/init-project/initProject.ts#L513

If the `templateName` variable is the name that is being sent to the telemetry service, you may get around this by adding an alias "quickstart" for the "clean" template here:
https://github.com/sanity-io/sanity/blob/b14622ed44899ad6da7bb0fd1d46f56c0186a04b/packages/@sanity/cli/src/actions/init-project/templates/index.ts#L9
LIke so:
```ts
const templates: Record<string, ProjectTemplate | undefined> = {
  blog,
  clean,
  'get-started': getStartedTemplate,
  moviedb,
  shopify,
  'shopify-online-storefront': shopifyOnline,
  quickstart: clean // New alias here
}
```

File: packages/@sanity/cli/templates/quickstart/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Sanity Clean Content Studio
```
Comment (RostiMelk) [inline packages/@sanity/cli/templates/quickstart/README.md:1]: Should we change this title as well while we're at it?

Comment (joneidejohnsen) [inline packages/@sanity/cli/templates/quickstart/README.md:1]:
```suggestion
# Sanity Studio
```

File: packages/@sanity/cli/templates/quickstart/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+Congratulations, you have now installed the Sanity Content Studio, an open source real-time content editing environment connected to the Sanity backend.
```
Comment (joneidejohnsen) [inline packages/@sanity/cli/templates/quickstart/README.md:3]:
```suggestion
Congratulations, you have now installed Sanity Studio, an open source real-time content editing environment connected to the Sanity Content Lake backend.
```

**Review by joneidejohnsen — CHANGES_REQUESTED** [review]
Suggested copy changes

**Review by jtpetty — APPROVED** [review]

**Review by joneidejohnsen — APPROVED** [review]

**Review by drewlyton — APPROVED** [review]

**Pull request merged**